ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -295,6 +307,11 @@ export class WalletDB {
       }
 
       await this.transactions.put([account.prefix, transactionHash], transactionValue, tx)
+      await this.timestampToTransactionHash.put(
+        [account.prefix, transactionValue.timestamp.getTime()],
```

Comment:
nit: above you use `transactionValue.timestamp.valueOf()` and here you use `getTime()`

Reply:
Changed to use `getTIme()` everywhere